README.md

```diff
@@ -171,6 +171,18 @@ import { h, render, Component } from 'preact';
 /** @jsx h */
 ```
 
+>
+> **For TypeScript:**
```

Comment:
The current readme has a section for TypeScript a few lines later. Can you move this there?